### ISSUE TITLE
[EXPLOIT]? Transferring all funds from cargo budget into ID so you can private purchase all content without access locks.

### ISSUE THREAD

**Issue by Feasel** [tgstation/tgstation#70134]
## Reproduction:

1. Make a private display stand that you buy from.
2. Mark it for ID that you want funds to be embezzled into.
3. Hide your normal ID in bag.
4. Wear Cargo Budget.
5. Purchase item for 1k credits.
6. Put item back in stand and close.
7. Purchase item for 1k credits.
8. Repeat till Cargo budget is empty.
9. Private Purchase all guns, batons, SM shards, or whatever you want! ENJOY!

**Comment by Baskett55** [comment]
This is called embezzling

**Comment by msgerbs** [comment]
Seems like the real mistake was making private purchases not have any access restrictions, since you need to already have access to the department budget to do this. Not sure this is an exploit so much as just bad design.

**Comment by gueba14** [comment]
was this fixed?

**Comment by msgerbs** [comment]
No, and I'm still not convinced there's anything to fix. This is working as designed.

**Comment by msgerbs** [comment]
@Fikou Does this need the exploit tag?

**Comment by Iamgoofball** [comment]
@msgerbs this is not working as designed, the cargo budget card is not supposed to be able to be withdrawn from, it's deposit only

**Comment by gueba14** [comment]
its fixed
i tried to recreate it in varios ways, could not do it

**Comment by SpaceSmithers** [comment]
![embezzlement](https://user-images.githubusercontent.com/105393050/215429800-972cac8a-7f52-40fe-a470-b7d359d07db8.png)
Works for me.
